src/goto-instrument/nondet_static.cpp

```diff
@@ -38,23 +38,27 @@ void nondet_static(
       const symbol_exprt &sym=to_symbol_expr(
           to_code_assign(instruction.code).lhs());
 
-      // is it a __CPROVER_* variable?
-      if(has_prefix(id2string(sym.get_identifier()), CPROVER_PREFIX))
-        continue;
-
       // static lifetime?
       if(!ns.lookup(sym.get_identifier()).is_static_lifetime)
         continue;
 
+      // should not be nondet-initialized, e.g., proprietary variables?
```

Comment:
Terminological nit-picking but ... what is a proprietary variable in an open source system?